modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -328,11 +282,7 @@ public static void ackReceived(UUID channelID, long messageID)
             if (log.isDebugEnabled()) {
                 log.debug("ack received for message id= " + messageID + " channelId= " + channelID);
             }
-            //This can be different from routing key in hierarchical topic case
-            AndesAckData andesAckData = AndesUtils.generateAndesAckMessage(channelID, messageID);
-            if(null == andesAckData) {
-                return;
-            }
+            AndesAckData andesAckData = new AndesAckData(channelID, messageID);
```

Comment:
Refactored. But this pattern is there in code in number of places. Suggest to modify in a separate PR.